packages/react-strict-dom/tests/html-test.native.js

```diff
@@ -114,7 +114,10 @@ describe('<html.*>', () => {
         rootColor: 'green'
       });
       const nestedTheme = css.createTheme(tokens, {
-        rootColor: 'blue'
+        rootColor: {
+          default: 'blue',
+          '@media (prefers-color-scheme: dark)': 'purple'
+        }
```

Comment:
Adjusting the fixture to surface regressions in handling of theme value objects. Previously this would not have set a color.